<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>280</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="pointsLabel">
      <property name="styleSheet">
       <string notr="true">font: 75 18pt &quot;C059&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Points:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLCDNumber" name="pointsLCD">
      <property name="styleSheet">
       <string notr="true">color: rgb(239, 41, 41);
background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>340</y>
     <width>84</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(239, 41, 41);</string>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>340</y>
     <width>84</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(239, 41, 41);</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="infoLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>200</y>
     <width>441</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 14pt &quot;C059&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>InfoBox</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>208</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="nameLabel">
      <property name="styleSheet">
       <string notr="true">font: 75 18pt &quot;C059&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Player name:         </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="levelLabel">
      <property name="styleSheet">
       <string notr="true">font: 75 18pt &quot;C059&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Level:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>289</x>
     <y>40</y>
     <width>241</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="nameLine"/>
    </item>
    <item>
     <widget class="QComboBox" name="levelBox">
      <property name="styleSheet">
       <string notr="true">selection-background-color: rgb(239, 41, 41);</string>
      </property>
      <item>
       <property name="text">
        <string>Instant Game</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Normal Game</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Long Game</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
